bokehjs/src/lib/models/annotations/arrow.ts

```diff
@@ -83,7 +83,7 @@ export class ArrowView extends AnnotationView {
     // Next we call .clip on all the arrow heads, inside an initial canvas sized
     // rect, to create an "inverted" clip region for the arrow heads
     ctx.beginPath()
-    const {x, y, width, height} = this.plot_view.layout.bbox
+    const {x, y, width, height} = this.plot_view.layout.inner_bbox
```

Comment:
I think it would be better to use `this.plot_view.frame.bbox`, though currently frame's outer box and plot's inner box are the same thing.

Reply:
Sure thing.

BTW is there any issue/PR about documenting all these fields? E.g. even with your comment it's hard to figure out what `layout` and `frame` truly are and when and why one should use one or the other.

Reply:
There is none currently and this is under another redesign right now to allow more flexible positioning within canvas (for 3.0), so I will hopefully document this when the design finally settles.